examples/django/components/django/mysite/mysite/settings.py

```diff
@@ -12,15 +12,15 @@
 DATABASES = {
     "default": {
         "ENGINE":
-        "django.db.backends.",  # Add 'postgresql_psycopg2', 'mysql', 'sqlite3' or 'oracle'.
+        "django.db.backends.",  # Add 'postgresql_psycopg2', 'mysql', ...
         "NAME": "",  # Or path to database file if using sqlite3.
         # The following settings are not used with sqlite3:
         "USER": "",
         "PASSWORD": "",
         "HOST":
-        "",  # Empty for localhost through domain sockets or '127.0.0.1' for localhost through TCP.
+        "",  # Empty for localhost through domain sockets or '127.0.0.1' ...
         "PORT": "",  # Set to empty string for default.
-    }}
+    }}  # noqa: E501 line too long
```

Comment:
```suggestion
    }}
```
This should not be necessary after the cut of the comments.